docker/README.md

```diff
@@ -2,55 +2,140 @@
+Before continuing, ensure you can successfully run the hello-world container.
+
+```
+docker run hello-world
 ```
 
+The output should include a `Hello from Docker!` message that will confirm everything is working with Docker and you can continue.
+
+If Docker is unable to pull the `hello-world:latest` image from Docker Hub, try disabling your VPN if you are using one.
+
+Linux users, note the lack of `sudo` before `docker run hello-world`. See the [Linux post-installation instructions](https://docs.docker.com/engine/install/linux-postinstall/) if you see the following error:
+```
+docker: permission denied while trying to connect to the Docker daemon socket at unix:///var/run/docker.sock: Post "http://%2Fvar%2Frun%2Fdocker.sock/v1.24/containers/create": dial unix /var/run/docker.sock: connect: permission denied.
+See 'docker run --help'.
+```
+
+### Cloning the Open Library repository
+
+If you have not yet forked and cloned the openlibrary repository, please see [Forking and Cloning the Open Library Repository](https://github.com/internetarchive/openlibrary/wiki/Git-Cheat-Sheet#forking-and-cloning-the-open-library-repository).
+
+Make sure you `git clone` openlibrary using `ssh` instead of `https` as git submodules (e.g. `infogami` and `acs`) may not fetch correctly otherwise. See [Modifying a repository wrongly cloned with https](https://github.com/internetarchive/openlibrary/wiki/Git-Cheat-Sheet#modifying-a-repository-wrongly-cloned-with-https-or-one-that-is-missing-the-infogami-module) if you cloned with `https`.
+
+## Setup Commands
+
+### For Windows Users Only
+
+Windows users should see [Fix line endings, symlinks, and git submodules](https://github.com/internetarchive/openlibrary/wiki/Git-Cheat-Sheet#fix-line-endings-symlinks-and-git-submodules-only-for-windows-users-not-using-a-linux-vm). Skipping those steps will likely prevent the containers from coming up.
+
 ### For Users of Macs Containing an M1 Chip
 
 Please use [Docker Desktop >= 4.3.0](https://docs.docker.com/desktop/mac/release-notes/) and make sure that Docker Desktop Preferences `General / Use Docker Compose V2` is checked so that it is no longer required to install Rosetta 2 to run Docker.
 
 If you are experiencing issues building JS, you may need to increase the RAM available to Docker. The defaults of 2GB ram and 1GB Swap are not enough. We recommend requirements of 4GB ram and 2GB swap. This resolved the error message of `Killed` when running `build-assets`.
 
 ### For All Users
-All commands are from the project root directory:
+All commands are from the project root directory, where `docker-compose.yml` is (i.e. `path/to/your/forked/and/cloned/openlibrary`):
 
-```bash
-# build images
-docker-compose build # 15+ min (Win10Home/Dec 2018)
+#### Build the images
+```
+docker compose build
+```
+This can take from a few minutes to up to 15+ on older hardware. If for some reason the build fails, it's worth running again, as sometimes downloads time out.
```

Comment:
```suggestion
This can take from a few minutes to more than 15 on older hardware. If for some reason the build fails, it's worth running again, as sometimes downloads time out.
```